lib/openapi_parser/schema_validators/minimum_maximum.rb

```diff
@@ -0,0 +1,18 @@
+class OpenAPIParser::SchemaValidator
+  module MinimumMaximum
+    # check enum value by schema
+    # @param [Object] value
+    # @param [OpenAPIParser::Schemas::Schema] schema
+    def check_minimum_maximum(value, schema)
+      include_min_max = schema.minimum || schema.exclusiveMinimum || schema.maximum || schema.exclusiveMaximum
+      return [value, nil] unless include_min_max
+
+      reference = schema.object_reference
+      return [nil, OpenAPIParser::LessThanMinimum.new(value, reference)] if schema.minimum && value < schema.minimum
+      return [nil, OpenAPIParser::LessThanExclusiveMinimum.new(value, reference)] if schema.exclusiveMinimum && value <= schema.exclusiveMinimum
```

Comment:
The OpenAPI 3 use exclusiveXXX properties is defined by JSON Schema Validation 00.
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#properties

In JSON Schema Validation, exclusiveXXX is defined as boolean value.
And when the exclusiveMinimum is true we should check `value < schema.minimum`, when it's false or not specified we should check `value <= schema.minimum`

```
   If "exclusiveMinimum" is true, then a numeric instance SHOULD NOT be
   equal to the value specified in "minimum".  If "exclusiveMinimum" is
   false (or not specified), then a numeric instance MAY be equal to the
   value of "minimum".
```
https://tools.ietf.org/html/draft-wright-json-schema-validation-00#section-5.3

So I think it's wrong validation.

Reply:
noted! sorry i didnt pay attention to the details. Fixed it! thanks @ota42y :)